Notifier-service/src/models/conference.ts

```diff
@@ -71,33 +71,55 @@ export class ConferenceModelI extends ConferenceModel{
     /*
         TO-DO
     */
-    async getConferences(offset:number , range:number):Promise<ConferenceDocument[] | null> {
-        let result = this.makeQuery((model) => {
-            return new Promise<ConferenceDocument[] | null>( (resolve , reject) => {
-                model.find({}).limit(range).exec((err , res) => {
-                    if(!err){
+    async getConferences(offset: number, range: number): Promise<ConferenceDocument[] | null> {
+
+        let result = this.model.then(model => {
```

Comment:
* Use makeQuery function ... the bove function has the logger activity and error handling already written in